src/vcpkg/commands.dependinfo.cpp

```diff
@@ -84,15 +84,17 @@ namespace vcpkg::Commands::DependInfo
 
         constexpr StringLiteral OPTION_DOT = "dot";
         constexpr StringLiteral OPTION_DGML = "dgml";
+        constexpr StringLiteral OPTION_MERMAID = "mermaid";
         constexpr StringLiteral OPTION_SHOW_DEPTH = "show-depth";
         constexpr StringLiteral OPTION_MAX_RECURSE = "max-recurse";
         constexpr StringLiteral OPTION_SORT = "sort";
 
         constexpr int NO_RECURSE_LIMIT_VALUE = -1;
 
-        constexpr std::array<CommandSwitch, 3> DEPEND_SWITCHES = {
+        constexpr std::array<CommandSwitch, 4> DEPEND_SWITCHES = {
             {{OPTION_DOT, []() { return msg::format(msgCmdDependInfoOptDot); }},
              {OPTION_DGML, []() { return msg::format(msgCmdDependInfoOptDGML); }},
+             {OPTION_MERMAID, []() { return msg::format(msgCmdDependInfoOptMermaid); }},
```

Comment:
I think you need to add validation that only one of these gets used?

Alternately, add `--format=<dot, or dgml, or mermaid>` and deprecate the old ones to make it clear that only one can be selected?